flow-typed/npm/jest_v23.x.x.js

```diff
@@ -227,6 +228,11 @@ type JestJQueryMatchersType = {
     toHaveDescendantWithText(sel: string, text: string | RegExp): void,
 };
 
+// TODO(WEB-902): create flow types for jest-enzyme
```

Comment:
I brought this task into this sprint b/c we want people to use jest-enzyme matchers when writing tests and they can't do that if flow is going to complain about them.